bokehjs/src/coffee/renderer/annotation/label.coffee

```diff
@@ -0,0 +1,112 @@
+_ = require "underscore"
+HasParent = require "../../common/has_parent"
+PlotWidget = require "../../common/plot_widget"
+properties = require "../../common/properties"
+
+class LabelView extends PlotWidget
+  initialize: (options) ->
+    super(options)
+    @text_props = new properties.Text({obj: @model, prefix: 'label_'})
+    @fill_props = new properties.Fill({obj: @model, prefix: 'background_'})
+    @line_props = new properties.Line({obj: @model, prefix: 'border_'})
+    @angle_props = new properties.Angle({obj: @model, attr: 'angle'})
```

Comment:
The initialize method for the Angle property is written differently than the Line/Text/Fill props and doesn't have a 'prefix' attr.